### PR TITLE
Feature/jwtodd

### PR DESCRIPTION
**Pull request by jwtodd**
'wasabis.sh package' builds app and ui rpm and deb files

### CODE REVIEW

**Comment by jwtodd** [comment]
@vinodtalati let's do this :)

File: bin/wasabi.sh [hunk 1]
```diff
@@ -173,7 +182,7 @@ resource() {
 }
 
 stop() {
-  ./bin/container.sh stop${1:+:$1}
+  ./bin/congit tainer.sh stop${1:+:$1}
```
Comment (vinodtalati) [inline bin/wasabi.sh]: @jwtodd  should this be container.sh?

**Comment by vinodtalati** [comment]
Hi James this looks good. Please see my comment for the possible typo. Other than that it looks good. Lets worry about the merge with my changes after this is through.

File: bin/wasabi.sh (hunk 1 above)
Comment (jwtodd) [inline bin/wasabi.sh]: ouch yes ... how did that happen ... good call

**Comment by jwtodd** [comment]
@vinodtalati merged ... let's get your PR merge ready :)

i will help as needed.

**Pull request merged**